hoomd/mpcd/tune.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2009-2023 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+r"""MPCD tuning operations.
+
+These operations will affect the performance of MPCD simulations but not
+their correctness.
+
+"""
+
+import hoomd
+from hoomd.mpcd import _mpcd
+from hoomd.operation import Tuner
+
+
+class ParticleSorter(Tuner):
+    r"""MPCD particle sorter.
+
+    Args:
+        trigger (hoomd.trigger.trigger_like): Number of integration steps
+            between sorting.
```

Comment:
```suggestion
        trigger (hoomd.trigger.trigger_like): Select the timesteps on which to
            sort.
```
Or other more generic language. A `trigger_like` may be a number, but can also be any `Trigger` subclass.